client/js/jsonedit.js

```diff
@@ -1808,21 +1839,38 @@ function jeSet(text, dontFocus) {
 // Replace ###SELECT ME### in JSON string in jeStateNow by the string given in replaceBy,
 // display the results in the text area by calling jeSet, and select the replaced text by calling jeSelect.
 function jeSetAndSelect(replaceBy, insideString) {
+
+  const emptyBrackets = replaceBy && ((Array.isArray(replaceBy) && replaceBy.length==0) || (typeof replaceBy === 'object' && Object.keys(replaceBy).length === 0)); // ###SELECT ME### should be replaced by [] or {}
+  const dollar = replaceBy && replaceBy == '${}'; // ###SELECT ME### should be replaced by ${} (and this will be in a string)
```

Comment:
Here the `replaceBy && ` should be unnecessary though, right?

Reply:
Fixed